src/reservoir/Reservoir.ts

```diff
@@ -1,9 +1,7 @@
 import {
   createClient,
   paths,
-  // TODO: remove this. See https://github.com/reservoirprotocol/reservoir-kit/pull/418
```

Comment:
No need to update package?

Reply:
Nope, since we have version `^` in the version number